docs/custom-flows/overview.mdx

```diff
@@ -56,7 +56,7 @@ Once a user is identified, they need to prove their identity. This is the proces
 
 ### Factor two verification (optional)
 
-This step only applies to users that have turned on two-factor authentication, also known as [multifactor authentication](/docs/custom-flows/mfa), for their user.
+This step only applies to users that have turned on two-factor authentication, also known as [multifactor authentication](/docs/custom-flows/email-password-mfa), for their user.
 
 When one form of verification isn't enough, trust two! Forcing two different verification steps vastly increases the security of your account. The most common setup a user will have to protect their account is using a password as their first kind of verification and a `phone_code` as their second kind of verification.
```

Comment:
```suggestion
Forcing two different verification steps vastly increases the security of your account. The most common setup a user will have to protect their account is using a password as their first kind of verification and a `phone_code` as their second kind of verification.
```
I know you didn't alter this line, but let's just cut the first sentence in this paragraph as it's just fluff.